billing_agreements.go

```diff
@@ -9,18 +9,16 @@ import (
 // Endpoint: POST /v1/billing-agreements/agreement-tokens
 func (c *Client) CreateBillingAgreementToken(
 	ctx context.Context,
-	name string,
 	description string,
-	startDate string,
+	shippingAddress *ShippingAddress,
```

Comment:
plase don't break backwards compatibility, this function was already released in a version. If you have used a CreateBillingRequest struct as an argument in the first place this would not have been a breaking change.